sync/course-db.js

```diff
@@ -1105,8 +1105,13 @@ async function validateAssessment(assessment, questions) {
   // Because of how Homework-type assessments work, we don't allow
   // real-time grading to be disabled for them.
   const allowRealTimeGrading = _.get(assessment, 'allowRealTimeGrading', true);
-  if (!allowRealTimeGrading && assessment.type === 'Homework') {
-    errors.push(`Real-time grading cannot be disabled for Homework-type assessments`);
+  if (assessment.type === 'Homework') {
+    if (!allowRealTimeGrading) {
+      errors.push(`Real-time grading cannot be disabled for Homework-type assessments`);
+    }
+    if (assessment.multipleInstance) {
+      errors.push(`Homework cannot have multiple instances`);
```

Comment:
This makes the error message a little more consistent with the others and hopefully speeds up resolution be referring to the specific option:
```suggestion
      errors.push(`"multipleInstance" cannot be used for Homework-type assessments`);
```